be2-scala/docs/README.md

```diff
@@ -284,8 +293,8 @@ Here's an example (shamefully stolen from `MessageHandler.scala`) showing the po
 ```scala
 val askWrite = dbActor ? DbActor.Write(rpcMessage.getParamsChannel, m)
 askWrite.transformWith {
-  case Success(_) => Future(Left(rpcMessage))
-  case _ => Future(Right(PipelineError(ErrorCodes.SERVER_ERROR.id, s"dbAskWrite failed : could not write message $message", rpcMessage.id)))
+  case Success(_) => Future(Right(rpcMessage))
+  case _ => Future(Left(PipelineError(ErrorCodes.SERVER_ERROR.id, s"dbAskWrite failed : could not write message $message", rpcMessage.id)))
```

Comment:
Is probably fine but just asking to make sure this Left and Right swap is intended? :)

Reply:
Indeed, we swapped the Left/Right to make them follow the scala convention :))